### PR TITLE
Fixing js minimization and removing sourcemaps in production

### PR DESCRIPTION
**Pull request by jfairchild**

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -114,29 +114,24 @@ gulp.task('scripts', gulp.series(() =>
...
+    .pipe(gulp.dest('./dist/scripts/'))
 ));
 
 // Scan your HTML for assets & optimize them
 gulp.task('html', gulp.series(() => {
   return gulp.src('app/**/*.html')
     .pipe($.useref({
       searchPath: '{.tmp,app}',
-      noAssets: true
+      noAssets: false
```
Comment (jfairchild) [inline gulpfile.babel.js:134]: i don't think this is doing anything for us.

File: gulpfile.babel.js [hunk 2]
```diff
@@ -114,29 +114,24 @@ gulp.task('scripts', gulp.series(() =>
...
     .pipe($.newer('.tmp/scripts'))
-    .pipe($.sourcemaps.init())
     .pipe($.babel())
-    .pipe($.sourcemaps.write())
     .pipe(gulp.dest('.tmp/scripts'))
     .pipe($.concat('main.min.js'))
     .pipe($.uglify({ preserveComments: 'some' }))
-    // Output files
     .pipe($.size({ title: 'scripts' }))
-    .pipe($.sourcemaps.write('.'))
```
Comment (jfairchild) [inline gulpfile.babel.js:130]: not sure why this was running twice, but we don't need it for production deploys.

File: .gitignore [hunk 3]
```diff
@@ -5,3 +5,4 @@ app.yaml
 .DS_Store
 .publish/
 .vscode/
+*.sh
```
Comment (wdoug) [inline .gitignore:8]: Why ignore all shell scripts?

File: README.md [hunk 4]
```diff
@@ -14,8 +14,8 @@
 - [Useful commands](docs/commands.md)
 - [Description of files in this directory](docs/file-appendix.md)
 
-Message Count
+Conversation Count
 ---
 
-1. Use the API Gateway endpoint to get the lastest value.
```
Comment (wdoug) [inline README.md:20]: Haha, I like `lastest` as a word

File: app/scripts/config.js [hunk 5]
```diff
@@ -18,7 +18,7 @@
     },
     conversations: {
       // count must be an integer
-      fallbackConversationCount: 4873330,
+      fallbackConversationCount: 5469087,
```
Comment (wdoug) [inline app/scripts/config.js:21]: We could probably update this again now

File: .gitignore (hunk 3 above)
Comment (jfairchild) [reply .gitignore:8]: i have a local deploy script i don't want checked in.

File: gulpfile.babel.js [hunk 6]
```diff
@@ -114,29 +114,23 @@ gulp.task('scripts', gulp.series(() =>
     './app/scripts/wst.js',
     './app/scripts/config.js',
     './app/scripts/lib/*.js',
-    // './node_modules/carousel-js/dist/carousel.js', TODOCAROUSEL
     './app/scripts/sections/*.js',
     './app/scripts/main.js'
-  ])
+  ], {base: './app/scripts/'})
```
[the remark this answers is not in the document]
Comment (jfairchild) [reply gulpfile.babel.js:119]: I had updated gulp in a previous PR. There was an issue with changing the relative paths of the js files if the `base` was not provided.

File: app/scripts/config.js [hunk 7]
```diff
@@ -18,7 +18,7 @@
     },
     conversations: {
       // count must be an integer
-      fallbackConversationCount: 4873330,
+      fallbackConversationCount: 5680775,
```
Comment (jfairchild) [inline app/scripts/config.js:21]: updated 👍

File: .gitignore (hunk 3 above)
Comment (wdoug) [reply .gitignore:8]: You could have just not committed that one script or specifically ignore it or a specific directory. It seems weird to me to just ignore all shell scripts in this repo.

**Review by wdoug — APPROVED** [review]
I'm not a huge fan of just ignoring all files ending in `.sh`, other than that this seems okay to me

**Pull request merged**